include/glow/Importer/ProtobufLoader.h

```diff
@@ -149,9 +149,17 @@ class ProtobufLoader {
 
   virtual ~ProtobufLoader();
 
-  /// \returns the output of the network. This is usually the result of the last
+  /// \returns the single final output of the network. The function assumes
+  /// there is only one output, verified via assertion. For image
+  /// classification, this single final output is usually the result of the last
   /// softmax or regression layer.
-  SaveNode *getRoot() { return root_; }
+  SaveNode *getSingleOutput() {
+    assert(outputsByName_.size() == 1);
+    return outputsByName_.begin()->second;
+  }
+
+  /// \returns the SaveNode for the external output with \p name.
+  SaveNode *getOutputByName(llvm::StringRef name) const;
```

Comment:
We should add a precondition (\pre) that the output with this name must exist or fix the implementation to return nullptr when it doesn't.

Reply:
Done